package.json

```diff
@@ -26,12 +26,13 @@
     "stage:venia": "cd packages/venia-concept && npm start; cd - >/dev/null",
     "test": "jest",
     "test:ci": "npm run -s test -- -i --json --outputFile=test-results.json",
-    "test:debug": "node --inspect-brk node_modules/.bin/jest -i",
+    "test:debug": "node --inspect-brk node_modules/.bin/jest -i --runInBand --no-cache --watch",
```

Comment:
The `-i` is already short for `--runInBand`, but `--watch` turns `--runInBand` off. I recommend changing this to just add `--no-cache`.

Reply:
Is the `--watch` command the reason jest wouldn't stop on `debugger` breakpoints? :man_facepalming: